x-pack/plugins/apm/public/components/app/TraceOverview/TraceList.tsx

```diff
@@ -32,23 +33,24 @@ interface Props {
 
 const traceListColumns: ITableColumn[] = [
   {
-    field: 'name',
+    field: 'sample',
     name: 'Name',
     width: '40%',
     sortable: true,
-    render: (name: string, transactionGroup: ITransactionGroup) => (
-      <TooltipOverlay content={formatString(name)}>
-        <StyledTraceLink transactionGroup={transactionGroup}>
-          {formatString(name)}
+    render: (transaction: Transaction) => (
+      <TooltipOverlay content={formatString(transaction.transaction.name)}>
+        <StyledTraceLink transaction={transaction}>
+          {formatString(transaction.transaction.name)}
         </StyledTraceLink>
       </TooltipOverlay>
     )
   },
   {
-    field: 'serviceName',
+    field: 'sample',
```

Comment:
There are now two fields called `sample`?

Reply:
Apart from that, I like the `Transaction` approach 👍

Reply:
Yeah I think the EUI basic table wants top level keys as the "field", so I am grabbing sample which I know is a `Transaction` and just grabbing what I need from it inside the render function.

Reply:
Ah, ofc. For one sec I thought that `field` had to be unique, so they would overwrite each other. 
nvm! :)